script.js

```diff
@@ -34,10 +34,17 @@ const addItemToList = () => {
 listContainer.addEventListener("click", (e) => {
   if(e.target.tagName.toLowerCase() === deleteItemTag) {
     e.target.parentElement.remove();
-    storeData()
   } else if (e.target.tagName.toLowerCase() === inputItemTag) {
     e.target.toggleAttribute('checked');
-    storeData();
+  }
+  storeData();
+});
+
+window.addEventListener("keydown", (e) => {
+  if (e.key === "Enter" && inputField.value !== "") {
+    addItemToList();
+  } else if (e.key === "Escape" && inputField.value !== "") {
```

Comment:
When we meet condition twice we extract it.

⚠️For example, instead of doing 
```
if (conditionA && conditionB) {
  // do some
}

if (conditionA && conditionC) {
  // do another
} 
```

✅Do this
```
if (conditionA) {
  if (conditionB) {
    // do some
  }

  if (conditionC) {
   // do another
  }
}
```

✅Or if `conditionA` wraps entire function body, there is another better option:
```
if (!conditionA) {
  return;
}

if (conditionB) {
  // do some
}

if (conditionC) {
 // do another
}
```

Reply:
Fixed)